### PR TITLE
Fmod extension

### PR DESCRIPTION
**Pull request by nassimberrada**

### CODE REVIEW

File: ivy/array/extensions.py [hunk 1]
```diff
@@ -341,3 +341,43 @@ def heaviside(
...
+        Returns
+        -------
+        ret
+            Array with element-wise remainder of divisions.
+
+        Examples
+        --------
+        >>> x1 = ivy.array([2, 3, 4])
+        >>> x2 = ivy.array([1, 5, 2])
+        >>> ivy.fmod(x1, x2)
```
Comment (Ishticode) [inline ivy/array/extensions.py:375]: should be instance method use here

File: ivy/functional/backends/numpy/extensions.py [hunk 2]
```diff
@@ -297,3 +297,25 @@ def heaviside(
...
+) -> np.ndarray:
+    return np.fmod(
+        x1,
+        x2,
+        out=None,
+        where=True,
+        casting="same_kind",
+        order="K",
+        dtype=None,
+        subok=True,
```
Comment (Ishticode) [inline ivy/functional/backends/numpy/extensions.py:317]: No need for these args
```python
        where=True,
        casting="same_kind",
        order="K",
        dtype=None,
        subok=True,
```

**Review by Ishticode — CHANGES_REQUESTED** [review]
Minor changes then I will link this with an issue and merge....

**Review by Ishticode — CHANGES_REQUESTED** [review]
This was almost ready but due to recent merging of the other PR i think there is more conflicts. Also there is `median` and `ndenumerate` functions added/modfied? is that intentional?

**Comment by nassimberrada** [comment]
I don't think i worked on those, probably someone else added those extensions ?
I solved the conflicts, here's the link to the corresponding issue fmod #5793

**Review by Ishticode — APPROVED** [review]

**Pull request merged**